packages/rest/src/rest-http-error.ts

```diff
@@ -51,11 +51,15 @@ export namespace RestHttpErrors {
 
   export const INVALID_REQUEST_BODY_MESSAGE =
     'The request body is invalid. See error object `details` property for more info.';
-  export function invalidRequestBody(): HttpErrors.HttpError {
+
+  export function invalidRequestBody(
+    details: ValidationErrorDetails[],
```

Comment:
would there be backward compatibility issue for adding a parameter?

Reply:
Good catch! 

In the strict interpretation, this is a breaking change because existing callers of `invalidRequestBody` must be updated. However, I consider `invalidRequestBody` as an internal helper that's not a part of our public API, therefore I think it's ok to make this change in a semver-minor release.